lib/single_cov.rb

```diff
@@ -76,6 +76,23 @@ def assert_tested(files: glob('{app,lib}/**/*.rb'), tests: default_tests, untest
       end
     end
 
+    def refute_complete_coverage_drop(tests: default_tests, currently_complete: [], currently_complete_file: "")
```

Comment:
they both test for the same, but are opposite of each other,
so the code be a single method with this structure ?
```
def assert_full_coverage(expected)
  actual = file read + regex
  missing = expected - actual
  assert missing.emtpy?, "something about file being removed"
  extra = actual - expected
  assert extra.emtpy?, "something about file being added"
```

Reply:
Feedback integrated!  Thank you <3